test/acceptor_SUITE.erl

```diff
@@ -1618,3 +1674,6 @@ do_os_supports_local_sockets() ->
 
 do_tempname() ->
 	list_to_binary(lists:droplast(os:cmd("mktemp -u"))).
+
+do_get_sockopts(Config) ->
```

Comment:
I think `ct_helper` has a function for something like this, `ct_helper:config/2`.

Reply:
Ok, I changed